### ISSUE TITLE
Feature Request: Per monitor/display settings

### ISSUE THREAD

**Issue by NEXT-JP** [dwarvesf/hidden#232]
It would be very cool to have different settings when plugged into an external monitor. When plugged into my external monitor, I have plenty of room for extra icons and don't need them hidden. I understand however that this will then clutter the smaller built in display, as I'm guessing the two screens cannot have different menu bars.

**Comment by dufu1991** [comment]
Yes!!! I hope to be able to set different automatic folding methods on different displays, such as closing the folding when the external display is very large, and opening the folding when the MacBook Pro built-in display is opened. It would be the perfect setting.
